release/release.go

```diff
@@ -135,6 +135,10 @@ func isWSL() bool {
 	return false
 }
 
+// PreseedMode states whether snapd is running in a presseding mode
+// where some functions and subsystems should alter their behavior.
+var PreseedMode bool // TODO: handle via devicestate, similar to DeviceContext.
```

Comment:
I have a mild preference to make this a function PreseedMode()  to help slightly the transition later, but not sure it changes a lot

Reply:
Done, no strong opinion either way.